app/controllers/iep_documents_controller.rb

```diff
@@ -0,0 +1,37 @@
+class IepDocumentsController < ApplicationController
+
+  before_action :find_iep_document, :authorize!
+
+  def show
+    @file_name = @iep_document.file_name
+    @bucket_name = ENV['AWS_S3_IEP_BUCKET']
+
+    object = s3.get_object({
```

Comment:
Does this need some kind of credentials from an ENV variable to be able to read from the bucket?